METRICS.md

```diff
@@ -15,132 +15,132 @@ By default it is `[server, nas, client]`.
-| [request, disconnect, server, $NAME, $IP, $PORT, total, undefined, undefined, gauge]              | histogram |
-| [request, disconnect, server, $NAME, $IP, $PORT, total, undefined, undefined, counter]            | counter   |
-| [response, total ,server, $NAME, $IP, $PORT, total, undefined, undefined, counter]                | counter   |
-| [response, access, server, $NAME, $IP, $PORT, total, undefined, undefined, counter]               | counter   |
-| [response, accounting, server, $NAME, $IP, $PORT, total, undefined, undefined, counter]           | counter   |
-| [response, access_accept, server, $NAME, $IP, $PORT, total, undefined, undefined, counter]        | counter   |
-| [response, access_reject, server, $NAME, $IP, $PORT, total, undefined, undefined, counter]        | counter   |
-| [response, access_challenge, server, $NAME, $IP, $PORT, total, undefined, undefined, counter]     | counter   |
-| [response, disconnect_ack, server, $NAME, $IP, $PORT, total, undefined, undefined, counter]       | counter   |
-| [response, disconnect_nak, server, $NAME, $IP, $PORT, total, undefined, undefined, counter]       | counter   |
-| [response, coa_ack, server, $NAME, $IP, $PORT, total, undefined, undefined, counter]              | counter   |
-| [response, coa_nak, server, $NAME, $IP, $PORT, total, undefined, undefined, counter]              | counter   |
-| [request, pending, server, $NAME, $IP, $PORT, total, undefined, undefined, counter]               | counter   |
-| [time, last_request, server, $NAME, $IP, $PORT, total, undefined, undefined, ticks]               | gauge     |
-| [time, since_last_request, server, $NAME, $IP, $PORT, total, undefined, undefined, ticks]         | gauge     |
-| [time, last_reset, server, $NAME, $IP, $PORT, total, undefined, undefined, ticks]                 | gauge     |
-| [time, last_config_reset, server, $NAME, $IP, $PORT, total, undefined, undefined, ticks]          | gauge     |
-| [time, up, server, $NAME, $IP, $PORT, total, undefined, undefined, ticks]                         | gauge     |
+_All metrics start with `eradius_radius_` prefix and the prefix is not included into table to save space._
+
+| Metric                                                             | Labels                  | Type      |
+| -------------------------------------------------------------------|-------------------------|-----------|
+| request_invalid_server_total_undefined_undefined_gauge             | [$NAME, $IP, $PORT]     | histogram |
+| request_invalid_server_total_undefined_undefined_counter           | [$NAME, $IP, $PORT]     | counter   |
+| request_malformed_server_total_undefined_undefined_gauge           | [$NAME, $IP, $PORT]     | histogram |
+| request_malformed_server_total_undefined_undefined_counter         | [$NAME, $IP, $PORT]     | counter   |
+| request_dropped_server_total_undefined_undefined_gauge             | [$NAME, $IP, $PORT]     | histogram |
```

Comment:
I appreciate the effort to keep the existing names, but prometheus has it's own idea what metric names should look like and I think we should follow their lead (@mgumz what's you take on this?)

see: https://prometheus.io/docs/practices/naming/